github-orgs.yml

```diff
@@ -53,6 +54,9 @@ getsentry:
       - 'sentry-react-native'
       - 'sentry-unity'
       - 'sentry-dart'
+      - 'sentry-go'
+      - 'sentry-unreal'
+      - 'sentry-elixir'
```

Comment:
@hubertdeng123 Go, Elixir and Unreal belong to the web team and are T2 and T3. See https://www.notion.so/sentry/Hover-display-interactions-are-potentially-confusing-41980d94706746cf82fed0ae86da6e3e

Electron is also T2, not sure we should have added them here.

Reply:
We've decided to expand triage beyond just T1 repos. See ticket here:
https://github.com/getsentry/team-ospo/issues/118